moveit_planners/ompl/ompl_interface/include/moveit/ompl_interface/parameterization/joint_space/constrained_planning_state_space.h

```diff
@@ -0,0 +1,72 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2012, Willow Garage, Inc.
```

Comment:
Use your copyright here too

Reply:
I see in most other files it's a company or institution (with some exceptions). Is it ok to just put my name there, or Picknik, or my current employer. I can ask it in our legal department if necessary.

Reply:
You are not a PickNik employee. You are paid by Google, but not a Google employee. I am not a lawyer, but I think that means you own the copyright in this case.

Reply:
It turned out that I had to refuse the Google payment for tax reasons, otherwise things became very complicated. Fortunately, it is so closely related to my Ph.D. that I'm still employed by KU Leuven. I'll ask them before it gets merged to make sure I do it right.

Reply:
In many universities (at least in the US), the intellectual property you develop as a PhD student belongs to your university, so I'd guess if you're doing this as part of your KU Leuven job then the copyright also belongs to them. Please check.

Reply:
I checked it with the legal advisor and indeed I need to fill in "KU Leuven".

Reply:
Did you also confirm that you can release it under the BSD license? :)